src/SDKs/PrivateDns/Management.PrivateDns/Microsoft.Azure.Management.PrivateDns.csproj

```diff
@@ -0,0 +1,22 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+	<!-- Please do not move/edit code below this line -->
+	<Import Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.reference.props'))" />
+	<!-- Please do not move/edit code below this line -->
+
+	<PropertyGroup>
+		<PackageId>Microsoft.Azure.Management.PrivateDns</PackageId>
+		<Description>Provides management capabilities for Azure Private DNS.</Description>
+		<Version>0.8.0-preview</Version>
+		<AssemblyName>Microsoft.Azure.Management.PrivateDns</AssemblyName>
+		<PackageTags>Microsoft Azure Private DNS;Private DNS;DNS</PackageTags>
+		<PackageReleaseNotes>Initial release.</PackageReleaseNotes>
```

Comment:
Could you make this a little more descriptive. Example [here](https://github.com/Azure/azure-sdk-for-net/blob/40248f466aa063bd90f16dae10d06b9b7317e2d0/src/SDKs/Blueprint/Management.Blueprint/Microsoft.Azure.Management.Blueprint.csproj#L14)